doc/supportedPlatformVerions.md

```diff
@@ -247,6 +247,8 @@
 
 ### ruby
 
+- 3.0.0
```

Comment:
Nit: should we add these two versions after 2.*? Also, it looks like we already have 3.0.0 below in the list.